autoload/ale/handlers/prettier.vim

```diff
@@ -0,0 +1,49 @@
+
+" Author: tunnckoCore (Charlike Mike Reagent) <mameto2011@gmail.com>
+" Description: Integration between Prettier and ESLint.
+
+" Here we use `prettier-eslint` intetionally,
+" because from v4 it is direct mirror of `prettier` - mimics
+" it's flags and etc.
+
+let g:ale_javascript_prettier_executable =
+\   get(g:, 'ale_javascript_prettier_executable', 'prettier-eslint')
+
+let g:ale_javascript_prettier_options =
+\   get(g:, 'ale_javascript_prettier_options', '')
+
+function! ale#handlers#prettier#GetExecutable(buffer) abort
+    if ale#Var(a:buffer, 'javascript_prettier_use_global')
```

Comment:
The problem came from when the handler for `eslint` has been loaded, but not the linter, because the variable was being initialised in the linter file by mistake. You will have copy and pasted the bug into this file. I have fixed that for `eslint` now.

You'll need to initialize `g:ale_javascript_prettier_use_global` to `0`.